pdc_client/__init__.py

```diff
@@ -203,10 +205,21 @@ def get_paged(self, res, **kwargs):
         """
         if self.page_size is not None:
             kwargs['page_size'] = self.page_size
-
+            page_size = self.page_size
             if self.page_size <= 0:
                 # If page_size <= 0, pagination will be disable.
                 return res(**kwargs)
+        else:
+            page_size = 20
+        if self.page is not None:
+            allinfo = res(page=self.page, **kwargs)
+            all_count = int(allinfo["count"])
+            if all_count % page_size > 0:
+                total = (all_count / page_size) + 1
+            else:
+                total = all_count / page_size
+            print("Total %d pages, here is the %d page, the count is %s." % (total, self.page, all_count))
+            return allinfo['results']
```

Comment:
In this get_paged function it just can return the results, like the yield codes:
`yield response['results']`
Because after this function, pdc will list the data by some columns in codes with the returned results.